### PR TITLE
CTR-269: UI changes as per FE UX review

### PR DESCRIPTION
**Pull request by vinnicombe**

### CODE REVIEW

File: test/views/behaviours/StringViewBehaviours.scala [hunk 1]
```diff
@@ -43,7 +42,6 @@ trait StringViewBehaviours extends QuestionViewBehaviours[String] {
 
         "contain a prefix for the value" in {
```
Comment (garethgooch) [inline test/views/behaviours/StringViewBehaviours.scala]: Is this testing anything?

Comment (vinnicombe) [reply test/views/behaviours/StringViewBehaviours.scala]: Good spot, removed.

File: test/views/HowMuchCarBenefitsViewSpec.scala [hunk 2]
```diff
@@ -49,7 +49,6 @@ class HowMuchCarBenefitsViewSpec extends StringViewBehaviours with MockitoSugar
       expectedFormAction = routes.HowMuchCarBenefitsController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchCarBenefitsViewSpec.scala]: I think these args can be removed, as the view no longer states the tax year in the question, its been moved to the secondary header.

File: test/views/DetailsOfEmploymentOrPensionViewSpec.scala [hunk 3]
```diff
@@ -52,7 +52,6 @@ class DetailsOfEmploymentOrPensionViewSpec extends StringViewBehaviours with Moc
       expectedFormAction = routes.DetailsOfEmploymentOrPensionController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = Some("detailsOfEmploymentOrPension.hintText"),
       expectedHintKeyLine2 = None,
-      expectedPrefix = None,
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/DetailsOfEmploymentOrPensionViewSpec.scala]: I think these args can be removed, as the view no longer states the tax year in the question, its been moved to the secondary header.

File: test/views/HowMuchBankInterestViewSpec.scala [hunk 4]
```diff
@@ -49,7 +49,6 @@ class HowMuchBankInterestViewSpec extends StringViewBehaviours with MockitoSugar
       expectedFormAction = routes.HowMuchBankInterestController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchBankInterestViewSpec.scala]: args remove

File: test/views/HowMuchCarBenefitsViewSpec.scala (hunk 2 above)
Comment (Pascoe121) [inline test/views/HowMuchCarBenefitsViewSpec.scala]: args remove

File: test/views/HowMuchForeignIncomeViewSpec.scala [hunk 5]
```diff
@@ -49,7 +49,6 @@ class HowMuchForeignIncomeViewSpec extends StringViewBehaviours with MockitoSuga
       expectedFormAction = routes.HowMuchForeignIncomeController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchForeignIncomeViewSpec.scala]: args remove

File: test/views/HowMuchFuelBenefitViewSpec.scala [hunk 6]
```diff
@@ -49,7 +49,6 @@ class HowMuchFuelBenefitViewSpec extends StringViewBehaviours with MockitoSugar
       expectedFormAction = routes.HowMuchFuelBenefitController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchFuelBenefitViewSpec.scala]: args remove

File: test/views/HowMuchInvestmentOrDividendViewSpec.scala [hunk 7]
```diff
@@ -49,7 +49,6 @@ class HowMuchInvestmentOrDividendViewSpec extends StringViewBehaviours with Mock
       expectedFormAction = routes.AnyOtherCompanyBenefitsController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchInvestmentOrDividendViewSpec.scala]: args remove

File: test/views/HowMuchMedicalBenefitsViewSpec.scala [hunk 8]
```diff
@@ -49,7 +49,6 @@ class HowMuchMedicalBenefitsViewSpec extends StringViewBehaviours with MockitoSu
       expectedFormAction = routes.HowMuchMedicalBenefitsController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchMedicalBenefitsViewSpec.scala]: args remove

File: test/views/HowMuchRentalIncomeViewSpec.scala [hunk 9]
```diff
@@ -49,7 +49,6 @@ class HowMuchRentalIncomeViewSpec extends StringViewBehaviours with MockitoSugar
       expectedFormAction = routes.HowMuchRentalIncomeController.onSubmit(NormalMode).url,
       expectedHintKeyLine1 = None,
       expectedHintKeyLine2 = None,
-      expectedPrefix = Some(messages("global.poundSign")),
       args = taxYear.asString(messages)
```
Comment (Pascoe121) [inline test/views/HowMuchRentalIncomeViewSpec.scala]: args remove

**Pull request merged**